### DISCUSSION TITLE
Appbar feature

### DISCUSSION THREAD

**Discussion by Cfun1** [CommunityToolkit/Maui#257] [New Feature Discussions]
Hello, is the appbar feature still on the table? https://github.com/xamarin/XamarinCommunityToolkit/issues/545

**Comment by Cfun1** [comment]
CC @brminnick @jfversluis @jsuarezruiz

**Reply by TheCodeTraveler** [reply]
Hi @Cfun1!

We’re always open to new features! We don’t currently have a [Proposal open for App Bar](https://github.com/CommunityToolkit/Maui/projects/1), but we’d be happy to add it if you put together a spec and an API design!

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
